test/assert.ts

```diff
@@ -14,6 +14,12 @@
  * limitations under the License.
  */
 
+declare namespace NodeJS {
+    interface Global {
+        assert: Chai.Assert;
```

Comment:
this interface augmentation is necessary to allow assigning `global.assert` on line 25. I'm not exactly sure why we need line 25 though; I can look into it.